Sources/App/Core/Search.swift

```diff
@@ -138,16 +137,20 @@ enum Search {
         // constants
         let emptyString = SQLLiteral.string("")
         let contains = SQLRaw("~*")
+        let tsrankDelimiter = SQLBinaryExpression(ts_rank(vector: tsvector, query: tsquery),
+                                                  .greaterThanOrEqual,
+                                                  SQLLiteral.numeric("0.05"))
```

Comment:
I think I can see what this is doing in terms of the SQL being created, but I'm not completely sure how this is working. It looks like it's setting a bool (or some other type) based on whether the `ts_rank` score is >= 0.05. Does that mean it's saying something like "yes, this is a good match"?

Reply:
It's the logic I described over in the issue: https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/2072#issuecomment-1277584969

The discriminator says: this has exact word matches or not.

Reply:
Ah, apologies! I hadn't received a notification of the issue updates. Looks great 👍